app/assess/routes.py

```diff
@@ -93,7 +93,12 @@ def display_sub_criteria(
             )
         )
 
-    fund = get_fund(Config.COF_FUND_ID)
+    assessor_task_list_metadata = get_assessor_task_list_state(application_id)
```

Comment:
Im wondering if we need to make a get fund method thats takes the application id instead of having you go to get_assessor_task_list_state

Reply:
We have the fund and round IDs when we load the dashboard page (eg. assessor_dashboard/nstf/r2) so could we just pass these in thru hidden form fields when we load the other routes and then use those to directly retrieve the fund name when we need it?

Reply:
Hmm, I don’t know if it’s worth worrying about this too much imo.  It’s just a bit of data and a get request, personal preference would be to have it reliable and encapsulated behind the scenes - if it becomes a performance issue in the future (which I doubt) then we can maybe change it then? 

I don’t think using a hidden field would work for GET requests, and even then it seems a bit off having to use a hidden field just to pass some data, users could also malform that if passed in through the UI form too.